tests/test_concurrency.py

```diff
@@ -94,7 +94,7 @@ def process():
     with (tmpdir / "mptest.py").open("w") as f:
         f.write(module)
     env = dict(os.environ)
-    env["PYTHONPATH"] = tmpdir + os.pathsep + env.get("PYTHONPATH", "")
+    env["PYTHONPATH"] = str(tmpdir + os.pathsep + env.get("PYTHONPATH", ""))
     # TODO: debug this. Importing c module fails on travis in this scenario
```

Comment:
Ha ha, the above might be the reason for this failure :D

Reply:
...nope, it doesn't. https://travis-ci.org/github/psycopg/psycopg3/jobs/746070487

Reply:
Yeah I've always found it a bit of a nuisance that `CalledProcessError` doesn't include stdout and stderr in its string representation. I know it might be quite lengthy, but it's a pain to have to add explicit output of `CalledProcessError.stdout/stderr` to work out what is going on in cases like this.